gridgencore.py

```diff
@@ -280,7 +280,13 @@ def __init__(self, axis, grid, key: str, val):
         else:
             self.title = grid.proc_variables(val.get("title"))
             self.description = grid.proc_variables(val.get("description"))
-            self.skip = (str(grid.proc_variables(val.get("skip")))).lower() == "true"
+            self.skip = False
+            self.skipList: bool | dict = val.get("skip")
```

Comment:
need snake_case.
Also, match the format of surrounding code - why is this the only line that has a type specification on it? Leave it off for this PR, if you think they're helpful to have make a followup PR adding them to at least entire sections at once so there's not inconsistency within a code block.